.github/labeler.yml

```diff
@@ -14,10 +14,26 @@
   - "doc/**/*"
   - "**/*.rst"
 
-"CI-libmodem-test":
+"CI-iot-samples-test":
   - "samples/nrf9160/**/*"
+  - "include/modem/**/*"
+  - "include/net/coap_utils.h"
+  - "include/net/download_client.h"
+  - "include/net/multicell_location.h"
+  - "include/secure_services.h"
+  - "lib/at_cmd/**/*"
+  - "lib/at_cmd_parser/**/*"
+  - "lib/at_host/**/*"
+  - "lib/at_monitor/**/*"
+  - "lib/at_notif/**/*"
   - "lib/lte_link_control/**/*"
-  - "lib/nrf_modem_lib/*"
+  - "lib_modem_key_mgmt/**/*"
+  - "lib/multicell_location/**/*"
+  - "lib/nrf_modem_lib/**/*"
+  - "lib/pdn/**/*"
+  - "lib/sms/**/*"
+  - "subsys/net/lib/coap_utils/**/*"
+  - "subsys/net/lib/download_client/**/*"
```

Comment:
for the lwm2m samples we should also add lib/bin/lwm2m_carrier to this list

Reply:
fixed